specs/Spec.Agile.HPKE.fst

```diff
@@ -180,8 +180,9 @@ val encap:
   Tot (option (key_kem_s cs & key_dh_public_s cs))
 
 #restart-solver
-#set-options "--z3rlimit 600 --fuel 0 --ifuel 2"
+#set-options "--z3rlimit 100 --fuel 0 --ifuel 0"
 let encap cs skE pkR =
+  let _ = allow_inversion Spec.Agile.DH.algorithm in
```

Comment:
Great to see typechecking of this spec simplified

Reply:
🙇